tests/test_playbooks/users_role.yml

```diff
@@ -0,0 +1,76 @@
+---
+# The following sensitive variables can be specified in the server.yml
+# ldap_host:
+# ldap_account:
+# ldap_password:
+# ldap_base_dn:
+# ldap_groups_base:
```

Comment:
Not true anymore?

Can you add information about the steps that are necessary to run / record this test?

Reply:
I´ve added it to the server.example also. 
What do you mean exactly?

Reply:
I think you mentioned you had a ldap container.
If that's something the developer needs to supply in order to record this test, it should be mentioned here.

Example:
https://github.com/theforeman/foreman-ansible-modules/blob/develop/tests/test_playbooks/realm.yml#L1

Reply:
Ah ok added some information.